flask_testing/utils.py

```diff
@@ -32,7 +32,11 @@
     # Python 2 urlparse fallback
     from urlparse import urlparse, urljoin
 
-from werkzeug import cached_property
+try:
+    from werkzeug.utils import cached_property
+except ImportError:
```

Comment:
The `try/except` isn't necessary, `cached_property` has always been available from `werkzeug.utils`.

Reply:
Thanks. Fixed.